packages/material-ui-styles/src/withStyles.test.js

```diff
@@ -51,9 +51,9 @@ describe('withStyles', () => {
 
       const ref = React.createRef();
       mount(
-        <>
+        <React.Fragment>
```

Comment:
What should we do with the Fragment. Should we embrace `<>` everywhere, should only use React.Fragment or it doesn't matter 🤔.

Reply:
I think `React.Fragment` is required if fragment refs land. So maybe enforce that?